### ISSUE TITLE
Violation with implicit values of element e.g. `span` + `width`

### ISSUE THREAD

**Issue by robole** [kristerkari/stylelint-declaration-block-no-ignored-properties#73]
Currently the block below is not a violation:

```css
span {
  width: 50px;
}
```

It would  be nice to make it a violation the same as:

```css
span {
  display:inline;
  width: 50px;
}
```

Are you willing to take a PR? I could take a look at it to see how it might work

**Comment by kristerkari** [comment]
@robole Thanks! We can't report that as invalid, because we don't know if the element in question has other styles applied to it.

**Comment by kristerkari** [comment]
And to further clarify: the readme states:

> Disallow property values that are ignored due to another property value in the same rule.

The idea behind the rule is to only flag properties if there is another property in the same style block that makes it invalid.

**Comment by robole** [comment]
I will close the issue if it is beyond the scope of the rule.

**Comment by kristerkari** [comment]
@robole Thanks for opening the issue anyway. The suggestion was good, but the problem with it is that it can cause false negatives since the rules for styling one element might be split to multiple blocks.

If you have more ideas for how to improve the plugin let me know!
